src/TicketService.java

```diff
@@ -31,14 +34,33 @@ public static void main(String[] args) {
             System.out.println(limitedTicket);
 // Saving ticket price
             emptyTicket.toSaveTicketPrice(49.99);
-            System.out.println(emptyTicket.toStringTicketPrice());
+            System.out.println(emptyTicket.printTicketPrice());
 //Generating tickets
             generateTenTickets();
             //testing getTicketById
             System.out.println(getTicketById("ID1"));
             //testing getTicketsByStadiumSector
             List<Ticket> ticketsByStadiumSector = getTicketsByStadiumSector(Sector.A, tickets);
             System.out.println(ticketsByStadiumSector.size());
+            //testing shared() by phone and by phone and email
+            emptyTicket.shared("123-456-789");
+            fullTicket.shared("123-456-789","ticket@email.com");
+            //testing users polymorphism
+            Client client = new Client("Nice Client");
```

Comment:
try use User client = new Client(); the same for admin

Reply:
add public abstract void getTicket();
    public abstract void checkTicket(); 
    into User and done.